apps/admin-portal/src/components/roles/create-role-wizard/role-user-assign.tsx

```diff
@@ -49,6 +53,18 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
         getUsersList(limit, offset, filter, attribute, null)
             .then((response) => {
                 setUsersList(response.Resources);
+
+                if (assignedUsers && assignedUsers.length != 0) {
+                    const selectedUserList: UserBasicInterface[] = [];
+                    response.Resources.forEach(user => {
```

Comment:
Would be better to check if `Resources` exists in the response and it's an iterable array.

Reply:
fixed with commit 0482c99